File: Web/ApplicationLayer/ViewModels/MRetrievalFormDetails_View.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ApplicationLayer.ViewModels
{
    public class MRetrievalFormDetails_View
    {
        public int id;
        public string department_name;
        public int Needed;
        public int Actual;
        public int stationery;

        public MRetrievalFormDetails_View(int id, string department_name, int Needed, int Actual, int stationery)
        {
            this.id = id;
            this.department_name = department_name;
            this.Needed = Needed;
            this.Actual = Actual;
            this.stationery = stationery;
        }
    }
}

File: Mobile Clients/StoreClerk(Android)/LogicUniversity/src/entity/MRetrievalFormDetails_View.java
package entity;

import java.util.HashMap;

public class MRetrievalFormDetails_View extends HashMap<String, String>{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	public int id;
	public String department_name;
	public int Needed;
	public int Actual;
	public int stationery;
	
	public MRetrievalFormDetails_View(int id, String department_name,
			int needed, int actual, int stationery) {
	
		this.id = id;
		this.department_name = department_name;
		Needed = needed;
		Actual = actual;
		this.stationery = stationery;
		
		put("id", Integer.toString(id));
		put("department_name", department_name);
		put("Needed", Integer.toString(Needed));
		put("Actual",Integer.toString(Actual));
		put("stationery",Integer.toString(Actual));
	}
	
	
	
}
